README.md

```diff
@@ -1,17 +1,21 @@
-# Axel
+# Problema
 
+Soy un promotor inmobiliario que busca expandir su actividad en la ciudad de Almería, para ello requerimos de analizar los datos de viviendo y urbanismo de forma que seamos capaces de identificar las mejores zonas para llevar a cabo nuevas edificaciones, así como predecedir los costes y los potenciales beneficios y rendimiento que se pueden obtener a largo plazo, ya sea mediante la venta o alquier de las viviendas a construir.
```

Comment:
Estás planteando una solución: "analizar los datos de vivienda" en vez de el problema. Si no dices por qué se quieren analizar, cualquier análisis sería válido como respuesta, pero también totalmente inútil y no resolvería nada.